srcpkgs/hyperfine/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'hyperfine'
 pkgname=hyperfine
-version=1.5.0
+version=1.6.0
 revision=1
 build_style=cargo
 short_desc="Command-line benchmarking tool"
 maintainer="Wilson Birney <wpb@360scada.com>"
-license="MIT"
+license="MIT, Apache-2.0"
 homepage="https://github.com/sharkdp/hyperfine"
 changelog="https://github.com/sharkdp/hyperfine/releases"
 distfiles="https://github.com/sharkdp/hyperfine/archive/v${version}.tar.gz"
-checksum=d5183611348e696e579dda846cee92b7b78c8ef18c00bfb9b0a62d0a63034823
+checksum=0cb251453147421c0c807587123e733b32346b9756c8594018c637ec374a63f4
+nocross=Multiple errors from rustc when cross-compiling.
```

Comment:
please put the message in double quotes.

Reply:
Done, thanx :)